model_compression_toolkit/core/common/hessian/hessian_info_service.py

```diff
@@ -165,13 +172,16 @@ def fetch_hessian(self, request: HessianScoresRequest,
         target_nodes = [self._get_primary_node(n) for n in request.target_nodes]
         request = request.clone(target_nodes=target_nodes)
 
-        if force_compute:
-            res = self._compute_hessians(request, self.num_iterations_for_approximation, count_by_cache=False)
+        if request.compute_from_tensors and activation_tensors is not None:
```

Comment:
Maybe if compute_from_tensors is true but the tensors were not provided, raise an error?